administrator/language/de-DE/com_finder.ini

```
@@ -171,6 +177,15 @@ COM_FINDER_INDEXER_MESSAGE_COMPLETE="Der Indexierungsprozess ist abgeschlossen.
 COM_FINDER_INDEXER_MESSAGE_INIT="Die Indexierung wird initialisiert. Achtung, das Fenster nicht schließen."
 COM_FINDER_INDEXER_MESSAGE_OPTIMIZE="Die Indextabelle wird für bestmögliche Darstellung optimiert. Achtung, das Fenster nicht schließen."
 COM_FINDER_INDEXER_MESSAGE_RUNNING="Der Inhalt wird indexiert. Achtung, das Fenster nicht schließen."
+COM_FINDER_INDEXER_MSG_DEBUGGING_INDEXING="Fehlersuche bei Suchindex Indexierungs-Plugins"
+COM_FINDER_INDEXER_MSG_DEBUGGING_INDEXING_TEXT="Ein Suchindex-Plugin auswählen und eine ID für den Index angeben. Das Ergebnis des Plugins für diese ID wird dann im unteren Bereich angezeigt."
+COM_FINDER_INDEXER_OUTPUT_AREA_TITLE="Ausgabe"
+COM_FINDER_INDEXER_TOOLBAR_TITLE="Indexer: Debug-Modus"
```

```suggestion
COM_FINDER_INDEXER_TOOLBAR_TITLE="Indexierung: Debug-Modus"
```
siehe https://github.com/joomlagerman/joomla/blob/22ad65809534ff078f95afdec4c31cb366aa90bf/administrator/language/de-DE/com_finder.ini#L120